### PR TITLE
Better error message when .claspignore is incorrect

### PR DESCRIPTION
**Pull request by tbenbrahim**
Issue #54 

CLA on file

### CODE REVIEW

**Comment by tbenbrahim** [comment]
Another alternative would be implementing `clasp status`, that would show  what files would be pushed by `clasp push`. This would be helpful in testing the validity of .claspignore, as is done with git

File: index.ts [hunk 1]
```diff
@@ -668,6 +669,10 @@ commander
                   error.errors.map((err: any) => {
                     console.error(err.message);
                   });
+                  console.error(LOG.FILES_TO_PUSH);
+                   nonIgnoredFilePaths.map((filePath) => {
```
Comment (grant) [inline index.ts]: Looks like there's an extra space here. Can you remove it?
Looks good otherwise, thanks!

**Comment by grant** [comment]
@tbenbrahim Can you fix small error in the GitHub editor?
I'll merge the PR after. Thanks!

**Comment by tbenbrahim** [comment]
@grant  done

**Comment by grant** [comment]
@tbenbrahim Awesome, thanks!
Will be updated in the next version of `clasp`.

**Pull request merged**